plugins/web/src/server_command.cpp

```diff
@@ -277,15 +278,43 @@ auto server_command(const vast::invocation& inv, caf::actor_system& system)
       return request->create_response(restinio::status_temporary_redirect())
         .append_header(restinio::http_field::server, "VAST")
         .append_header_date_field()
-        .append_header(restinio::http_field::location, "/api/v0/status")
+        .append_header(restinio::http_field::location, "/status.html")
```

Comment:
I think this is not what we want. Routing is being fully managed in the frontend. So this is creating a `/status.html` route instead of the better-looking `status` (and is also messing with the frontend router). The backend should just serve the webroot directory at some endpoint, everything after that is frontend's responsibility.